docs/en/resources/sharing-data.md

```diff
@@ -1,41 +1,42 @@
-# Data
+# Sharing Data
 
-Places to access collections of GTFS and other transit and multimodal data
+Places to access collections of GTFS and other transit and multimodal data.
 
-## 3rd party GTFS URL directories
+### 3rd party GTFS URL directories
 - [The Mobility Database](https://mobilitydatabase.org/) - JSON and CSV files [on GitHub](https://github.com/MobilityData/mobility-database-catalogs) that is a repository of 2000+ mobility datasets across the world. Contains contents of OpenMobilityData/TransitFeeds.com.
```

Comment:
@Sergiodero I would suggest upping the number to 4000 and add "from over 75 countries", latest stats from https://mobilitydatabase.org/.